metriki-core/Cargo.toml

```diff
@@ -15,6 +15,7 @@ readme = "../README.md"
 crossbeam-utils = "0.8"
 hdrhistogram = { version = "7", default-features = false, features = [] }
 lazy_static = "1"
+dashmap = "4.0"
```

Comment:
The latest release of dashmap is `5.0`. Is there any reason we need to stick with `4.0`?

Reply:
nope, just do not know there is a new version T T